server/src/main/java/org/elasticsearch/action/admin/indices/shrink/PrepareReindexTargetAction.java

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+package org.elasticsearch.action.admin.indices.shrink;
+
+import org.elasticsearch.action.ActionType;
+
+public class PrepareReindexTargetAction extends ActionType<ResizeResponse> {
```

Comment:
I don't think this should extend `ActionType<ResizeResponse>` since it is not performing any kind of resize operation.

Reply:
Sure. Can I use `CreateIndexResponse` as it will be the same for cloning index? or Shall I create a new Response and Request class altogether.

Reply:
Yes, I think `CreateIndexResponse` would be appropriate here.